source/features/separate-draft-pr-button.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+function init(): void {
+	const previewForm = select('.new-pr-form')! as HTMLFormElement;
+
+	// Move the "Allow edits from maintainers" to its own line if it exists
+	const allowEditsCheckbox = select('.float-left', previewForm);
+	if (allowEditsCheckbox) {
+		allowEditsCheckbox.classList.remove('float-left');
+	}
+
+	const buttonBar = select('.flex-justify-end', previewForm)!;
+	const defaultButton = select('.BtnGroup', buttonBar)!;
+
+	defaultButton.hidden = true;
+
+	const menuItems = select.all('.select-menu-item', defaultButton);
+
+	for (const menuItem of menuItems) {
+		const title = select('.select-menu-item-heading', menuItem)!.textContent!.trim();
+		const description = select('.description', menuItem)!.textContent!.trim();
+		const radioButton = select('[type=radio]', menuItem)! as HTMLInputElement;
+
+		const classList = ['btn', 'ml-2', 'tooltipped', 'tooltipped-s'];
+		if (!title.toLowerCase().includes('draft')) {
+			classList.push('btn-primary');
+		}
+
+		const submitForm = (): void => {
+			radioButton.checked = true;
+			previewForm.submit();
+		};
```

Comment:
I don’t think all of this is necessary. Probably you can set the `name` and `value` to the button and that will be submitted instead of the radio. 

Also generally use more descriptive variable names because “menuItems” doesn’t tell me what they are.